### PR TITLE
fix: select all should be dispalyed if reconcilated is ON whatever (#CrowdFunding_DarkJeff_2021)

### PR DESCRIPTION
**Pull request by FHenry**
select all should be bdispalyed if reconsilated is ON

### CODE REVIEW

**Comment by FHenry** [comment]
@Darkjeff Croundfounding 2021

File: htdocs/compta/bank/bankentries_list.php [hunk 1]
```diff
@@ -1091,15 +1101,7 @@
...
-				print '<td class="center">';
-				print '<input type="checkbox" id="selectAll" title="'.dol_escape_htmltag($langs->trans("SelectAll")).'" />';
-				print ' <script type="text/javascript">
-						$("input#selectAll").change(function() {
-							$("input[type=checkbox][name^=rowid]").prop("checked", $(this).is(":checked"));
-						});
-						</script>';
-				print '</td>';
-				print '<td colspan="'.($tmpnbfieldafterbalance + 2).'">';
+				print '<td colspan="'.($tmpnbfieldafterbalance + 1).'">';
```
Comment (eldy) [inline htdocs/compta/bank/bankentries_list.php:1104]: 2 td You replaced 2 td into 1 but the second one
<td colspan="'.($tmpnbfieldafterbalance + 2).'">';
become
<td colspan="'.($tmpnbfieldafterbalance + 1).'">';

May be it should be  
<td colspan="'.($tmpnbfieldafterbalance + 3).'">';   ?

Comment (FHenry) [reply htdocs/compta/bank/bankentries_list.php]: Done

**Comment by Darkjeff** [comment]
we have a problem with existing num-releve and the select all
I test the new version ^^

**Comment by FHenry** [comment]
Correct fix already done in 13.0 branch

**Pull request closed without merging**